src/Npgsql/NpgsqlConnector.cs

```diff
@@ -1925,13 +1927,14 @@ public Task WaitAsync(CancellationToken cancellationToken)
 
         #region Supported features and PostgreSQL settings
 
-        bool SupportsCloseAll => ServerVersion >= new Version(8, 3, 0);
-        bool SupportsAdvisoryLocks => ServerVersion >= new Version(8, 2, 0);
-        bool SupportsDiscardSequences => ServerVersion >= new Version(9, 4, 0);
-        bool SupportsUnlisten => ServerVersion >= new Version(6, 4, 0) && !IsRedshift;
-        bool SupportsDiscardTemp => ServerVersion >= new Version(8, 3, 0);
-        bool SupportsDiscard => ServerVersion >= new Version(8, 3, 0); // Redshift is 8.0.2
-        internal bool SupportsRangeTypes => ServerVersion >= new Version(9, 2, 0);
+        bool SupportsCloseAll => ServerVersion >= new Version(8, 3, 0) && !IsCrateDB;
```

Comment:
These may belong on DatabaseInfo... As I said above let me think about the architecture a little bit.